backend/config.js

```diff
@@ -53,10 +53,10 @@ const apis = {
     timeoutSeconds: process.env.KEYWORKER_API_TIMEOUT_SECONDS || 30,
     ui_url: process.env.MANAGE_KEY_WORKERS_URL,
   },
-  complexityOfNeed: {
+  complexity: {
     url: process.env.COMPLEXITY_OF_NEED_URI || 'http://localhost:18091',
     timeoutSeconds: process.env.API_ENDPOINT_TIMEOUT_SECONDS || 10,
-    enabled: process.env.ENABLE_COMPLEXITY_API === 'true',
+    enabled_prisons: process.env.PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS,
```

Comment:
Hmm.. having this config in 2 separate services is error prone - never mind the fact that the complexity team probably have the same list somewhere else.

Also, we don't want to have to release the service just to add more prisons, but that is secondary.

Reply:
It's technically in three if you include the key worker API. Other than storing the config in some shared persisted store I'm not sure what else to do, do you have any suggestions?

Reply:
I will raise it with the tech team. I wasn't expecting any changes, just felt like i needed to comment on the problem?!